site.mk

```
@@ -39,18 +38,16 @@ GLUON_MULTIDOMAIN := 1
 # This is the Stable branch
 
 # Gluon Base Release
-DEFAULT_GLUON_RELEASE := v3.2
+DEFAULT_GLUON_RELEASE := v3.3
+
+# Development branch information
+GLUON_BRANCH ?= test
```

Ich dachte das wird eine `stable`?

```suggestion
GLUON_BRANCH ?= stable
```

---

Ist in der ToDo-Liste ja noch nicht abgehakt ;)

---

Fixed in 19d4cd1